src/main/java/aeromon/command/EditCommand.java

```diff
@@ -0,0 +1,52 @@
+package aeromon.command;
+
+import aeromon.AeromonException;
+import aeromon.Storage;
+import aeromon.Ui;
```

Comment:
Maybe the aeromo.Ui import can swap with the aermon.task imports? So that the package imports will be in alphabetical order 😉